### PR TITLE
Issue #829 Button text not visible while selecting the avatar in landscape mode resolved

### PR DESCRIPTION
**Pull request by harshitagupta30**
Fixes Issue #289 

@chhavip Please review! :)

### CODE REVIEW

**Comment by harshitagupta30** [comment]
Screenshots of updated UI

screenshot_2017-01-29-19-52-11
screenshot_2017-01-29-19-52-14

**Comment by aanchal07** [comment]
@harshitagupta30 Can you increase the size of the text "Continue" in the button? Also in the avatar screen can you place the continue button in alignment with the back button? Rest of the work is good.

**Comment by harshitagupta30** [comment]
@aanchal07 @chhavip I hope it looks fine now! :)
  
screenshot_2017-01-30-23-39-56
screenshot_2017-01-30-23-39-53

File: PowerUp/app/src/main/res/layout-land/avatar.xml [hunk 1]
```diff
@@ -77,7 +77,11 @@
         android:layout_alignTop="@+id/backButton"
         android:layout_toEndOf="@+id/hairView"
         android:layout_toRightOf="@+id/hairView"
-        android:background="@drawable/brown_button_background" />
+        android:background="@drawable/brown_button_background"
+        android:text="@string/continue_text"
+        android:textSize="@dimen/button_font"
+        android:layout_alignBottom="@+id/backButton"
+        android:layout_marginRight="90dp"/>
```
Comment (aanchal07) [inline PowerUp/app/src/main/res/layout-land/avatar.xml]: Remove hard coding.

**Comment by aanchal07** [comment]
@harshitagupta30  Also please squash your commits in one.

**Comment by harshitagupta30** [comment]
@aanchal07 I have made the required changes! Please review!

**Comment by aanchal07** [comment]
@harshitagupta30  Good work!
@chhavip Please check this and merge the pr.

**Comment by chhavip** [comment]
@aanchal07 Has this PR been run on device and checked? Will merge if it has been running okay

**Comment by aanchal07** [comment]
@chhavip Yes, I checked it on Genymotion emulator.

**Comment by anubhakushwaha** [comment]
@harshitagupta30 Please remove the conflicts so that this can be merged.

**Pull request closed without merging**